conftest.py

```diff
@@ -152,4 +153,12 @@ def reboot_sensor(make_valid_request):
     def inner():
         return make_valid_request(SensorMethod.REBOOT)
 
-    return inner
\ No newline at end of file
+    return inner
+
+
+@pytest.fixture
+def wait(make_valid_request):
```

Comment:
While there should be a function called `wait()` in the conftest, it's supposed to do something a bit different. Please take a look at the files attached to the homework, you'll find the code for this function there.